hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/WALEntrySinkFilter.java

```diff
@@ -51,9 +51,8 @@
   void init(AsyncConnection conn);
 
   /**
-   * @param table Table edit is destined for.
-   * @param writeTime Time at which the edit was created on the source.
+   * @param entry WALEntry.
    * @return True if we are to filter out the edit.
    */
-  boolean filter(TableName table, long writeTime);
+  boolean filter(WALEntry entry);
```

Comment:
Exposing the PB object here is the concern. We did lot of work for 2.0 to avoid this.

Reply:
Yes, what @anoopsjohn  said. We don't want PBs in our API.